exporter/sapmexporter/README.md

```diff
@@ -32,6 +33,16 @@ during final translation.  Intended to be used in tandem with identical configur
   User should calculate this as `num_seconds * requests_per_second` where:
     - `num_seconds` is the number of seconds to buffer in case of a backend outage
     - `requests_per_second` is the average number of requests per seconds.
+- `correlation`
+  - `enabled` (default = true): Whether to enable span/metric correlation.
+  - `endpoint` (default = ""): If set this is the endpoint where correlation API calls will be made. Otherwise it defaults to using `api_endpoint` value.
```

Comment:
Same issue with _endpoint_ term.

Reply:
The URL in the comment for `HTTPClientSettings` does show the full URL with path to the actual HTTP resource being POSTed to, so it seems ok there.